api/src/test/resources/io/strimzi/api/kafka/model/Kafka-with-jbod-storage-on-zookeeper.yaml

```diff
@@ -15,13 +15,13 @@ spec:
     storage:
       type: jbod
       volumes:
-        - id: 1
+        - id: 0
```

Comment:
It's under `volumes` so I would see `volumeId` as redundant. I think that it's clear a volume `id`. As opposite if we wanted to mean a broker id we have used `brokerId`.

Reply:
Yes, but else where we'll end up with some broker `id` in a place where the context is more broker-y. And at that point there's the potential for confusion as we have `id` meaning different things in different places.

Reply:
Why we should use `brokerId` in a context where the broker is clear and `id` is enough?

Reply:
Yeah, I'm not saying that we _definitely should_. I was merely wondering whether, ultimately, it would be clearer if we _consistently_ used `brokerId`, `volumeId`, `partitionId` etc, rather than relying on context all the time, or (worse) ending up with contexts where `id` really is ambiguous and then having to (inconsistently) use _<qualifier>_Id.